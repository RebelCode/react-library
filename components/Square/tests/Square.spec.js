import React from "react";
import { render, cleanup } from "@testing-library/react";
import { Square } from "../index";

afterEach(cleanup);

describe("Square", () => {
  const Comp = (
    <Square>
      <div>
        <p>
          Lorem Ipsum is simply dummy text of the printing and typesetting
          industry. Lorem Ipsum has been the industry's standard dummy text ever
          since the 1500s
        </p>
      </div>
    </Square>
  );

  it("should take a snapshot", () => {
    const { asFragment } = render(Comp);

    expect(asFragment(Comp)).toMatchSnapshot();
  });

  it("renders correctly", () => {
    const { queryByLabelText } = render(Comp);

    expect(queryByLabelText("Square")).toBeTruthy();
  });
});
