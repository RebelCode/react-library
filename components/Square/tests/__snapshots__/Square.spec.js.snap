// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Square should take a snapshot 1`] = `
<DocumentFragment>
  <div
    aria-label="Square"
    class="filler"
  >
    <div
      class="positioner"
    >
      <div>
        <p>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
